xml/System.Security.Cryptography.Pkcs/SignedCms.xml

```diff
@@ -720,12 +720,14 @@ The following permissions are required to access the signature key:
       </Parameters>
       <Docs>
         <param name="signer">A <see cref="T:System.Security.Cryptography.Pkcs.CmsSigner" /> object that represents the signer.</param>
-        <param name="silent">This parameter is not used. </param>
+        <param name="silent">.NET Core only: <see langword="true" to to request opening keys with PIN prompts disabled, where supported; otherwise <see langword="false">. In .NET Framework, this parameter is not used and a PIN prompt is always shown, if required.</param>
```

Comment:
```suggestion
        <param name="silent">.NET Core only: <see langword="true" /> to to request opening keys with PIN prompts disabled, where supported; otherwise <see langword="false" />. In .NET Framework, this parameter is not used and a PIN prompt is always shown, if required.</param>
```